compio-runtime/src/runtime/mod.rs

```diff
@@ -30,18 +29,32 @@ pub(crate) mod time;
 use crate::runtime::time::{TimerFuture, TimerRuntime};
 use crate::{runtime::op::OpFuture, BufResult};
 
+scoped_tls::scoped_thread_local!(static CURRENT_RUNTIME: Runtime);
+
 pub type JoinHandle<T> = Task<Result<T, Box<dyn Any + Send>>>;
 
-pub(crate) struct RuntimeInner {
+/// The async runtime of compio. It is a thread local runtime, and cannot be
```

Comment:
It can. Runtime is `Send`.

Reply:
I prefer not. Maybe I should add a phantom field...